gnome-initial-setup/pages/summary/gis-summary-page.ui

```diff
@@ -19,12 +19,81 @@
             <property name="valign">center</property>
             <property name="orientation">vertical</property>
             <property name="spacing">12</property>
-            <child type="center">
-              <object class="GtkImage" id="image">
+            <child>
+              <object class="GtkBox" id="summary_contents_box">
                 <property name="visible">True</property>
-                <property name="resource">/org/gnome/initial-setup/ready-to-go.svg</property>
-                <property name="width_request">96</property>
-                <property name="height_request">96</property>
+                <property name="halign">center</property>
+                <property name="valign">center</property>
+                <property name="orientation">vertical</property>
+                <property name="spacing">12</property>
+                <child type="center">
+                  <object class="GtkImage" id="image">
+                    <property name="visible">True</property>
+                    <property name="resource">/org/gnome/initial-setup/ready-to-go.svg</property>
+                    <property name="width_request">96</property>
+                    <property name="height_request">96</property>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkBox">
+                    <property name="visible">True</property>
+                    <property name="halign">center</property>
+                    <property name="valign">center</property>
+                    <property name="orientation">vertical</property>
+                    <child>
+                      <object class="GtkBox" id="warning_box">
+                        <property name="visible">True</property>
+                        <property name="halign">center</property>
+                        <property name="valign">center</property>
+                        <property name="orientation">horizontal</property>
+                        <property name="margin_start">24</property>
+                        <property name="margin_end">24</property>
+                        <property name="margin_bottom">6</property>
+                        <property name="spacing">6</property>
+                        <child>
+                          <object class="GtkImage" id="warning_icon">
+                            <property name="visible">True</property>
+                            <property name="can_focus">False</property>
+                            <property name="icon-name">dialog-warning-symbolic</property>
+                            <property name="pixel_size">24</property>
+                            <property name="valign">center</property>
+                            <property name="hexpand">True</property>
+                          </object>
+                        </child>
+                        <child>
+                          <object class="GtkLabel" id="tagline">
```

Comment:
Have you tested how the "all data will be lost" warning looks in live + hack and live + not-hack modes? It looks like you're moving it above the "Start using Endless" button. Why?

Reply:
I did! I think I had actually seen it above the button, but maybe I am wrong. I will double check.